src/redisjson.rs

```diff
@@ -216,6 +280,28 @@ impl RedisJSON {
         }
     }
 
+    pub fn get_memory<'a>(&'a self, path: &'a str) -> Result<usize, Error> {
+        let res = match self.get_doc(path)? {
+            Value::Null => 0,
+            Value::Bool(_v) => mem::size_of::<bool>(),
```

Comment:
It seems more readable to use `mem::size_of_val(...)` instead of manually coding the types.

Reply:
@gavrie I'm not sure how you can do it, the value is not given

Reply:
What I mean is this:

Instead of:
`Value::Bool(_v) => mem::size_of::<bool>()`

Do:
`Value::Bool(v) => mem::size_of_val(v)`

This is simple more readable since it avoid the need for the turbofish and explicit types.

Reply:
done